### ISSUE TITLE
Bug: Scholar of the Lost Trove cannot cast Adventure sides.

### ISSUE THREAD

**Issue by Susucre** [magefree/mage#12229]
[[Scholar of the Lost Trove]] should allow to cast the Adventure side of [[Twice Upon a Time]], but there is no option to do so.

**Comment by iamrm91** [comment]
I don't think that is supposed to occur. Their explanation on official page:

"While Beanstalk Giant is on the battlefield, just ignore the Adventure and all its text. While the card is in your hand, your graveyard, your library, even exile, it has only the characteristics of the creature, not the Adventure. For example, if you cast a spell that lets you search your library and find a creature card, you can find Beanstalk Giant. But if you cast a spell that lets you search your library and find an instant or sorcery card, you won't be able to find Fertile Footsteps."

https://magic.wizards.com/en/news/feature/throne-eldraine-mechanics-2019-09-04

**Comment by iamrm91** [comment]
I don't think that is supposed to occur. Quote from official page:

"While Beanstalk Giant is on the battlefield, just ignore the Adventure and all its text. While the card is in your hand, your graveyard, your library, even exile, it has only the characteristics of the creature, not the Adventure. For example, if you cast a spell that lets you search your library and find a creature card, you can find Beanstalk Giant. But if you cast a spell that lets you search your library and find an instant or sorcery card, you won't be able to find Fertile Footsteps."

https://magic.wizards.com/en/news/feature/throne-eldraine-mechanics-2019-09-04

This seems to mean the adventure side can't be targeted from graveyard unless they print a card that specifically says so.

**Comment by JayDi85** [comment]
@iamrm91 

“Allow cast” effects must be applied to all possible characteristics (normal and adventure versions). And it can allow only a specific part (e.g. creature + instant or only creature or only instant).

Details from https://boardgames.stackexchange.com/questions/48801/can-i-cast-adventures-from-my-graveyard


Related rules:

> Casting a card as an Adventure isn't casting it for an alternative cost. Effects that allow you to cast a spell for an alternative cost or without paying its mana cost may allow you to apply those to the Adventure.
(2019-10-04)

> 601.3e If a rule or effect states that only an alternative set of characteristics or a subset of characteristics are considered to determine if a card or copy of a card is legal to cast, those alternative characteristics replace the object’s characteristics prior to determining whether the player may begin to cast it.

Related rules example:
> [[Melek, Izzet Paragon]] says, in part, “You may cast the top card of your library if it’s an instant or sorcery card.” If you control Melek, Izzet Paragon and the top card of your library is [[Giant Killer]], an adventurer creature card whose Adventure is an instant named Chop Down, you may cast Chop Down but not Giant Killer. If instead you control [[Garruk’s Horde]] and the top card of your library is Giant Killer, you may cast Giant Killer but not Chop Down.

**Comment by JayDi85** [comment]
[[Melek, Izzet Paragon]] [[Giant Killer]] [[Garruk’s Horde]]

**Comment by iamrm91** [comment]
I stand corrected. Should I delete the post so they don't cause any confusion or unnecessary clutter?

**Comment by JayDi85** [comment]
Nope, it’s just a discussion. You can edit it and add line in the start like “That’s wrong, see next” or use `~~wrong~~` to write ~~wrong~~ part.
